src/systems/core/arena.js

```diff
@@ -670,14 +670,16 @@ AFRAME.registerSystem('arena-scene', {
 
             if (sceneOptions.physics) {
                 // physics system, build with cannon-js: https://github.com/c-frame/aframe-physics-system
-                import('../vendor/aframe-physics-system.min');
-                const physicsWait = setInterval(() => {
-                    // wait for physics system and static-body component to be registered, needs 15-30 ms
-                    if (AFRAME.components['static-body']) {
-                        clearInterval(physicsWait);
-                        document.getElementById('groundPlane').setAttribute('static-body', 'type', 'static');
-                    }
-                }, 10);
+                import('../vendor/aframe-physics-system.min').then(() => {
+                    const physicsWait = setInterval(() => {
+                        // wait for physics system and static-body component to be registered, needs 15-30 ms
+                        if (AFRAME.components['static-body']) {
+                            clearInterval(physicsWait);
+                            document.getElementById('groundPlane').setAttribute('static-body', 'type', 'static');
```

Comment:
I had some thoughts about leaving this out and having each user implement their own groundPlane, which might match how other systems like Unity handle it, but I think that is a discussion for another day.

Reply:
As above, can probably just ignore this pending a physics system change. There should be some event emit actually, when a component is fully loaded somewhere in AFRAME land